source/release-notes/backend.txt

```diff
@@ -13,6 +13,39 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240605:
+
+5 June 2024 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Atlas Events
+
+  - Apps with the Atlas Events feature flag will start to react to changes made in their linked Atlas clusters.
+
+  - On cluster deletion, Apps will delete datasources and all resources linked to that datasource for datasources connected to the deleted cluster.
+
+  - When the cluster is updated (instance size change), the next subsequent usage of a Mongo Client will have small 
+    cold start time caused by refreshing client connection details.
+
+- Improvements to Function Editing Experience: All Apps will now be able to use a new function editor with autocomplete for common functionalities and App Services context. This includes:
+
+  - A new button to allow formatting your functions' code
+
+  - A new clear and run button to replace the Clear result one
+  
+  - Context autocomplete
```

Comment:
```suggestion
- Improvements to Function editor. All Apps can now use the new function editor with autocomplete for common functionalities and App Services context. This includes:

  - A new :guilabel:`BUTTON NAME?` button to format function code.

  - A new :guilabel:`Clear and Run` button that replaces the :guilabel:`Clear Result` button.
  
  - Context autocomplete.
```
is the "Context autocomplete" bullet kind of redundant with the `All Apps can now use the new function editor with autocomplete for common functionalities and App Services context.` sentence?

Reply:
Thanks, fixed!